### ISSUE TITLE
How can I trained on ImageNet-A as IN and evaluate on ImageNet as OOD?

### ISSUE THREAD

**Issue by alirezaabdollahpour** [mlfoundations/wise-ft#15]
Hi ;)
How can I train on **ImageNet-A as the in-distribution (IN)** dataset and evaluate on I**mageNet as the out-of-distribution (OOD) dataset?**

**Comment by gabrielilharco** [comment]
Hi @alirezaabdollahpour. Unlike ImageNet, ImageNet-A doesn't have a training set (only a test set), so I don't think this would be possible

**Comment by alirezaabdollahpour** [comment]
> Hi @alirezaabdollahpour. Unlike ImageNet, ImageNet-A doesn't have a training set (only a test set), so I don't think this would be possible

Thank you, Gabriel, for your response. Can we assume that the test data represents the entire dataset and then split it into training and testing sets?

**Comment by gabrielilharco** [comment]
Sure, although I'd keep in mind that the dataset is quite small already, so splitting it like you suggested might lead to training difficulties and more noise in the evaluations

**Comment by alirezaabdollahpour** [comment]
Thanks
